src/icons/general/IconWorkflow.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconWorkflow = ({ className = '', color = '#999999', height = 32, title, width = 32 }: Props) => (
```

Comment:
could we swap the default here with `$nines` (or the closest BDL color `$bdl-neutral-03`)?